dev-games/tiled/tiled-1.11.0.ebuild

```
@@ -0,0 +1,97 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+inherit toolchain-funcs multiprocessing flag-o-matic python-single-r1 xdg
+
+DESCRIPTION="A general purpose tile map editor"
+HOMEPAGE="https://www.mapeditor.org/"
+SRC_URI="https://github.com/mapeditor/tiled/archive/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD BSD-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="python examples"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	app-arch/zstd:=
+	dev-qt/qtbase:6[X,opengl,widgets]
+	dev-qt/qtdeclarative:6
+	dev-qt/qtsvg:6
+	sys-libs/zlib
+	python? ( ${PYTHON_DEPS} )
+"
```

```suggestion
RDEPEND="
	app-arch/zstd:=
	dev-qt/qtbase:6[X,dbus,gui,network,opengl,widgets]
	dev-qt/qtdeclarative:6
	dev-qt/qtsvg:6
	sys-libs/zlib
	python? ( ${PYTHON_DEPS} )
"
```

https://github.com/search?q=repo%3Amapeditor%2Ftiled++%22Depends+%7B+name%3A%22&type=code